.github/workflows/fastlane_ci.yml

```
@@ -0,0 +1,66 @@
+      - develop
+    types: [closed]
+
+jobs:
+  fastlane:
+    name: Run Fastlane
+    runs-on: macos-latest
+    if: github.event.pull_request.merged == true
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)